step.yml

```diff
@@ -1,7 +1,44 @@
 title: Manage iOS Code Signing
-summary: TODO
+summary: Automatically manages your code signing files.
 description: |-
-  TODO
+  The **Manage iOS Code Signing** Step takes care of automatically code signing your project before it's built on Bitrise.
+  The Step:
+  - generates, updates and downloads the provisioning profiles needed for your iOS project.
+  - verifies if your project is registered with the App Store Connect. If it was not, the Step registers your project.
```

Comment:
On `App Store Connect` you manage your Apps in App Store, review App Analytics, Sales and Trends, manage Users and Access.

The actions we do with this Step are targeting the Apple Developer Site (some calls it Apple Developer Portal).

Reply:
It verifies and registers the project's Bundle IDs (not the project itself)